src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -777,22 +779,11 @@ public Bindings(ModelLoadContext ctx, TextLoader parent)
                 }
 
                 _slotNames = new VBuffer<DvText>[Infos.Length];
-                List<DvText> lines = null;
-                // If the loader has a header in the data file, try reading a new header.
-                if (parent.HasHeader)
-                    Cursor.GetSomeLines(parent._files, 2, ref lines);
```

Comment:
>s(parent._files,  [](start = 38, length = 17)

Scope can finally handle models with header?
Or I shouldn't be excited?

Reply:
I didn't specifically change it for this reason, but I think you might be right

---
In reply to: [212742415](https://github.com/dotnet/machinelearning/pull/723#discussion_r212742415) [](ancestors = 212742415)